x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -292,13 +292,19 @@ Writer options let you specify the separator characters used when writing messag
 * The starting transaction set control number (used only when no current value is found in the object store).
 * The default interchange usage indicator (ISA15), used when no value is present in the message data.
 * Whether to request interchange acknowledgments (ISA14).
-* Whether to use supplied values for control segment information (control numbers, in particular). The default behavior is to always generate control numbers when writing, this allows you to instead use values you have chosen.
+* Whether to use supplied values for control segment information (control numbers, in particular).
+
+The default behavior is to always generate control numbers when writing, this allows you to instead use values you have chosen.
```

Comment:
Starting here the bullet lists are not properly formatted and won't render correctly. Need to fix the bullet lists.